services/frontend/src/components/AnalysisVisualization.vue

```diff
@@ -1,20 +1,136 @@
 <template>
-  <div>
-    <!-- create graphs here -->
-    <!-- this is a mock -->
-    <img src="mock_files/mock_analysis_graph.png"/>
+  <div v-if="analisStared" class="sector flex-column-items-centered card"> Start Analysis before starting </div>
```

Comment:
"Start Analysis before starting" first of all it a bag english sentence. Second of all we were ment to write UI in polish, because we are creating a product for a polish client.